views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -296,7 +360,7 @@ define([
                 startForReal();
             })
             .catch(function(err) {
-                console.error(err);
+                console.error(err.message);
```

Comment:
Should not it be better to log the full error instead? We are loosing the error context here.

Reply:
No problem.